### DISCUSSION TITLE
Greyed out "History Rollback button" when using ApplicatonSet

### DISCUSSION THREAD

**Discussion by wofr** [argoproj/argo-cd#18380] [Q&A]
I do have the following setup.  

I've an "Application" (yaml) which does link to an "ApplicationSet" (yaml). 
The History button does only work on the "Application" but for all "Application" derived from the ApplicationSet the history button is greyed out.

The setup works totally fine, but of course it would be nice seeing the history of the apps from the ApplicationSet.   I don't know if this is a bug or just not possible to show the history due to my given setup. 

I'm using ArgoCD Version (v2.8.4+c279299)

**Comment by agaudreault** [comment]
Maybe it is a bug in the older version. You can see in the demo instance that History is available for applications created from appset in https://cd.apps.argoproj.io/applications/argocd/example.helm-hooks?resource=&rollback=0
